docs/features/spelling-and-grammar-checking.md

```diff
@@ -1,77 +1,85 @@
+The {@link features/language text part language} feature lets the user set different languages to different portions of the content. The spell and grammar check feature offers full support for a multilingual content. If the WProofreader language is set to Auto Detect (or the `auto` language is set in the configuration), the feature will automatically recognize the language for a given sentence and suggest spelling and grammar corrections specifically for that language, as shown in the demo above.
 
 {@img assets/img/spellcheck-dictionary.png 770 Setting the spell checker dictionary to auto.}
 
+## Check types
+
+WProofreader checks texts for spelling, grammar, and punctuation mistakes. The auto detect feature facilitates correction of multilingual texts. Users do not have to manually switch languages to proofread combo docs. Handy spelling autocorrect validates user texts on the fly. Autocomplete suggestions for English make the proofreading process fast and smooth. The predictive text can be accepted with a right <kbd>→</kbd> arrow key.
+
 ## Supported languages and dictionaries
 
 ### Language support
 
-The most popular languages used with WProofreader include: American English, British English, Canadian English, Canadian French, French, German, Italian, Greek, Spanish, Finnish, Danish, Dutch, Portuguese, Swedish, Ukrainian, Norwegian Bokmål, Brazilian Portuguese. There are, however, over 160 languages altogether, available for download from the WebSpellChecker site. Grammar checking is available for over 20 languages.
+The most popular languages used with WProofreader include: American English, Australian English, Arabic, Brazilian Portuguese, British English, Canadian English, Canadian French, Danish, Dutch, Finnish, French, German, Greek, Hebrew, Italian, Indonesian, Norwegian Bokmål, Norwegian Nynorsk, Portuguese, Spanish, Swedish, Turkish, and Ukrainian.
+
+There are, however, more languages available from the WebSpellChecker site. Grammar checking is available for over 20 languages.
 
-A recent addition to the software are AI-driven tools. Smart algorithms employed in the AI-based language dictionaries offer a far better checking quality, generating proofreading suggestions based on the context of the sentence. They provide more suitable suggestions that address mistakes with thrice the accuracy of traditional dictionaries. The AI-based support is currently available for English and German.
+The AI-driven tools approach for English, German, and Spanish is a recent addition to the software. It offers a far better checking quality, and generates proofreading suggestions based on the context of a sentence. It provides more suitable suggestions that address mistakes with 3 times the accuracy compared to a traditional mechanism. The AI-based support is currently available for English and German.
```

Comment:
So, is it English, German and Spanish (first sentence) or just English and German (last sentence) because this is not clear?

Reply:
The last sentence remained from the previous version. Thanks!